### DISCUSSION TITLE
v1.13.04

### DISCUSSION THREAD

**Discussion by dorkster** [flareteam/flare-game#903] [Announcements]
## Engine features

* Added 'bar_gfx' property to menus/enemy.txt for setting the "fill" image

## Engine fixes

* Fix crash when trying to read a non-existent file with INCLUDE
* Fix passive powers not being correctly re-applied after death
* Improve performance of WidgetListBox by caching all items instead of just the visible ones

<hr /><em>This discussion was created from the release <a href='https://github.com/flareteam/flare-game/releases/tag/v1.13.04'>v1.13.04</a>.</em>

**Comment by WithinAmnesia** [comment]
Cool!
